src/mainboard/raptor-cs/talos-2/Kconfig

```diff
@@ -5,6 +5,7 @@ if BOARD_RAPTOR_CS_TALOS_2
 config BOARD_SPECIFIC_OPTIONS
 	def_bool y
 	select CPU_POWER9
+	select CPU_IBM_POWER9
```

Comment:
CPU_POWER9 czy CPU_IBM_POWER9

Reply:
I have left only `CPU_IBM_POWER9`. I'm not sure about `src/cpu/power9/Kconfig`, `CPU_POWER9` is used there

Reply:
Maybe `CPU_IBM_POWER9` should be renamed to `SOC_IBM_POWER9` or something like that. We definitely need code from `src/cpu/power9` as well.

Reply:
I have renamed it to `SOC_IBM_POWER9`.